src/article/editor/EditorPackage.js

```diff
@@ -265,6 +266,9 @@ export default {
       nodeType: 'superscript',
       commandGroup: 'formatting'
     })
+    config.addCommand('add-custom-metadata-field', AddCustomMetadataFieldCommand, {
```

Comment:
I'd like to define commands in alphabetic order. Less git conflicts, and more clear how to do it.

Reply:
👍